argilla-server/src/argilla_server/security/authentication/oauth2/settings.py

```diff
@@ -83,7 +83,7 @@ def from_dict(cls, settings: dict) -> "OAuth2Settings":
 
     @classmethod
     def _build_workspaces(cls, settings: dict) -> List[AllowedWorkspace]:
-        allowed_workspaces = settings.pop(cls.ALLOWED_WORKSPACES_KEY) or []
+        allowed_workspaces = settings.pop(cls.ALLOWED_WORKSPACES_KEY, None) or []
```

Comment:
This is not the same?

```suggestion
        allowed_workspaces = settings.pop(cls.ALLOWED_WORKSPACES_KEY, [])
```

Reply:
Not really:
```python
{
   "allowed_workspaces": None
}

settings.pop(cls.ALLOWED_WORKSPACES_KEY, []
# None
settings.pop(cls.ALLOWED_WORKSPACES_KEY, None) or []
# []